content/blog/tidymodels-2021-q1/index.Rmd

```diff
@@ -0,0 +1,141 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2021-q1
+title: Catch up with tidymodels
+date: 2021-03-10
+author: Julia Silge
+description: >
+    Recent tidymodels releases offer new functions for easier model building and
+    resampling, along with a new package for resampling spatial data.
+
+photo:
+  url: https://unsplash.com/photos/4Zk45jNyQS4
+  author: Timo Wielink
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [package] 
+tags: [tidymodels, parsnip, rsample]
+---
+
+<!--
+TODO:
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+```{r include=FALSE}
+library(ggplot2)
+theme_set(theme_minimal())
+```
+
+
+There have quite a number of updates and new developments in the [tidymodels](https://www.tidymodels.org/) ecosystem since our [last blog post in December](https://www.tidyverse.org/blog/2020/12/finetune-0-0-1/)! Since that post, tidymodels maintainers have published seven CRAN releases of existing packages:
+
+- [broom](https://broom.tidymodels.org/news/)
+- [butcher](https://butcher.tidymodels.org/news/)
+- [embed](https://embed.tidymodels.org/news/)
+- [parsnip](https://parsnip.tidymodels.org/news/)
+- [rsample](https://rsample.tidymodels.org/news/)
+- [rules](https://rules.tidymodels.org/news/)
+- [tune](https://tune.tidymodels.org/news/)
+
+TODO: TUNE NEWS/PKGDOWN NOT YET UPDATED
+
+We purposefully write code in small, modular packages to make them easier to maintain (for us!) and use in production systems (for you!) but this does mean that sometimes any given package release can feel a bit minor. We haven't written much about these releases yet, but we want to make sure that folks can stay up-to-date with the changes and new features available. We plan to begin regular updates every three or four months here on the tidyverse blog summarizing what's happening lately in the tidymodels ecosystem overall, as well as the focused blog posts on more major new features that we've always written. The `NEWS` files are linked above for each package, but read on for some highlights that may interest you!
+
```

Comment:
Need to briefly list the major happenings to motivate reading the rest of the post. Probably better to put that before a mechanical listing of the updated packages.

Reply:
The `tune` website has been updated.